Css/aboutMe.css

```diff
@@ -0,0 +1,95 @@
+* {
+    padding: 0;
```

Comment:
global padding and margin should not be added in *, it's usually better to do it for the body tag